financial-connections/src/main/java/com/stripe/android/financialconnections/ui/theme/Type.kt

```diff
@@ -0,0 +1,88 @@
+@file:Suppress("MatchingDeclarationName")
+
+package com.stripe.android.financialconnections.ui.theme
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Immutable
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.tooling.preview.Preview
+
+@Immutable
+internal data class FinancialConnectionsTypography(
+    val subtitle: TextStyle,
+    val subtitleEmphasized: TextStyle,
+    val heading: TextStyle,
+    val subheading: TextStyle,
+    val kicker: TextStyle,
+    val body: TextStyle,
+    val bodyEmphasized: TextStyle,
+    val detail: TextStyle,
+    val detailEmphasized: TextStyle,
+    val caption: TextStyle,
+    val captionEmphasized: TextStyle,
+    val captionTight: TextStyle,
+    val captionTightEmphasized: TextStyle,
+)
+
+@Composable
```

Comment:
A lot of these previews would be great to make into screenshot tests. It looks like you're doing things like adding long strings etc, the work would be fairly minimal. Let me know if you need any help

Reply:
I'd love to have some screenshot testing in place! I'll let you know as soon as we have a few more components in place so we can have a meaningful set of testable components. Thanks!!!